lib/pbench/server/api/resources/__init__.py

```diff
@@ -969,7 +980,7 @@ def authorize(self, params: ApiParams) -> Optional[ApiAuthorization]:
                     role=self.operation,
                 )
             return None
-        elif self.authorization == API_AUTHORIZATION.USER_ACCESS:
+        elif self.authorization == ApiAuthorizationType.USER_ACCESS:
```

Comment:
Sigh. They're either isolated or tightly integrated with code I was changing anyway. It would have been a pain to merge at the point I decided to make the changes, and I really don't think it would have made the review significantly easier.

Reply:
I can go a lot faster when I know that I'm only looking for things which have changed name (or any other type of change which shouldn't alter behavior).  Having to make sure that I don't miss anything _between_ the name changes (or changes to use the changed names differently), slows down the scan.  (And, in fact, I ran out of gas before I could finish on my previous attempt...but I'm psyching myself up for my next assault.)

I wouldn't have thought that the merge would be that hard -- yeah, you would have conflicts where the rename affected code that you'd touched for your changes, but those should be easy to resolve, particularly if you do the name-change to your new code separately.